<launch>
<group ns="plain_map">
    <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/plain_map.yaml">
    </node>
  </group>
 <group ns="global_map">
  <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/ochang_map.yaml" >
  </node>
 </group>
  <group ns="region_map">
    <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/ochang_region_map.yaml">
    </node>
  </group>
  <group ns="signalStop_map">
    <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/ochang_signal_stop.yaml">
    </node>
  </group>
  <group ns="obs_map">
    <node name="map_server" pkg="map_server" type="map_server" args="$(find global_path_planner)/map_data/ochang_region_map.yaml">
    </node>
  </group>
  <node name="Sim_path_tracking" pkg="Sim_path_tracking" type="Sim_path_tracking" respawn="true">
    <param name="simORreal" value="real" type="string"/>
    <param name="velocity_gain" value="1.0" type="double"/>
  </node>
  <node name="globa_path_publisher" pkg="global_path_planner" type="ochang_globa_path_publisher.py" output="screen" args="ochangHalf" respawn="true" />
  <node name="velodyne_process" pkg="velodyne_process" type="velodyne_process" respawn="true"/>
  <node name="ochang_local_costmap" pkg="global_path_planner" type="ochang_local_costmap" respawn="true" output="screen" args="final_path">
    <rosparam file="$(find global_path_planner)/config/ochang_config.yaml" />
  </node>
  </launch>
